docs/dictionaries.md

```diff
@@ -149,7 +149,7 @@ this is a Neurobagel common data element.
 ## Age
 Neurobagel has a common data element for `"Age"` which describes a continuous column. To ensure age values are represented as floats in Neurobagel graphs, Neurobagel encodes the relevant "heuristic" describing the value format of a given age column. This heuristic, stored in the `Transformation` annotation, corresponds internally to a specific transformation that is used to convert the values to float ages.
 
-Possible heuristics are: `float`, `int`, `euro`, `bounded`, `range`, `iso8601`
+Possible heuristics are: `float`, `int`, `euro`, `bounded`
```

Comment:
```suggestion
Possible heuristics are: `float`, `int`, `euro`, `bounded`, `iso8061`
```
Like we said in the discussion: `iso8061` is something we want, but don't have yet everywhere (i.e. the annotation tool). So let's keep it in the docs, but make an issue in the annotation_tool repo to make sure this heuristic gets added.